proto/xudrpc.proto

```diff
@@ -102,6 +102,15 @@ service Xud {
     };
   }
 
+  /* Gets an address to deposit a given currency into the channel.
```

Comment:
Should this say "a channel" ? I also think "directly" would be helpful here as in "deposit a given currency directly into a/the channel"